distribution/lib/Standard/Test/0.0.0-dev/src/Extensions.enso

```diff
@@ -251,6 +258,16 @@ Error.should_succeed : Integer -> Any
 Error.should_succeed self frames_to_skip=0 =
     Test.fail_match_on_unexpected_error self 1+frames_to_skip
 
+## Handles an unexpected dataflow error.
+Error.should_be_a : Integer -> Any
+Error.should_be_a self frames_to_skip=0 =
+    Test.fail_match_on_unexpected_error self 1+frames_to_skip
+
+## Handles an unexpected dataflow error.
+Error.should_be_an : Integer -> Any
+Error.should_be_an self frames_to_skip=0 =
+    Test.fail_match_on_unexpected_error self 1+frames_to_skip
```

Comment:
I'd probably remove the `should_be_an` but can it please go through `should_be_a`

Reply:
Since we removed `is_an`, I think `should_be_an` should go to for consistency. I will remove it.